src/modules/pos/validator/components/EditStake/index.test.js

```diff
@@ -160,15 +160,15 @@ describe('EditStake', () => {
   });
 
   it('should render the edit stake modal', async () => {
-    const delegate = mockValidators.data[0];
+    const validator = mockValidators.data[0];
 
     useSentStakes.mockReturnValue({
       data: {
         ...mockSentStakes,
         data: {
           ...mockSentStakes.data,
           votes: mockSentStakes.data.votes.map((vote, index) =>
```

Comment:
@reyraa Assuming the data from the endpoint has been updated too, shouldn't this be `stakes: ...`?

Reply:
The mocks have not been updated. so for now we have to keep it this way.